lib/ui/painting/image_decoder_unittests.cc

```diff
@@ -5,13 +5,21 @@
 #include "flutter/common/task_runners.h"
 #include "flutter/fml/mapping.h"
 #include "flutter/fml/synchronization/waitable_event.h"
+#include "flutter/fml/trace_event.h"
 #include "flutter/lib/ui/painting/image_decoder.h"
 #include "flutter/testing/test_gl_surface.h"
 #include "flutter/testing/testing.h"
 #include "flutter/testing/thread_test.h"
 #include "third_party/skia/include/codec/SkCodec.h"
 
 namespace flutter {
+
+// Declaration of internal method implemented in image_decoder.cc.
+sk_sp<SkImage> ImageFromCompressedData(sk_sp<SkData> data,
```

Comment:
You can move this to image_decoder.h as a public static method and declare the method just once there.

Reply:
Happy either way, just thought to keep it under wraps since I'm also lifting linkage. Done